lessons/beginners/install-editor/vscode.md

```diff
@@ -0,0 +1,29 @@
+{% set editor_name = 'VSCODE' %} {% set editor_url = 'https://code.visualstudio.com' %} 
+{% extends lesson.slug + '/_base.md' %}
+
+{% block name_gen %} Instalace VSCode{% endblock %}
+{% block setup %}
+
+Editor VSCode or Microsoftu podobně jako Atom funguje out-of-the-box (nemusí se nic nastavovat).
```

Comment:
Please keep the text independent of the other editors. People reading this might not be familiar with Atom.

Reply:
ušetřil bych je i termínu "out-of-the-box"

Reply:
Pokud jde o editaci textu, tak souhlasím.
Pokud jde o psaní kódu v Pythonu, doporučil bych instalaci rozšíření pro jeho podporu v IDE.

Reply:
Ted upne nechapu co tim myslis. Je tam napsano instalace rozsireni pro vscode ne pro IDE